src/work-allocation/containers/task-list/task-list.component.html

```diff
@@ -4,8 +4,9 @@
   <!-- Column Headers and Cells -->
   <ng-container *ngFor="let field of fields" [cdkColumnDef]="field.name">
     <th class="govuk-table__header" [attr.aria-sort]="getColumnSortedSetting(field.sortName)" cdk-header-cell *cdkHeaderCellDef>
-      <button class="column-sort" *ngIf="field.columnLabel" type="button" [id]="'sort_by_' + field.sortName" data-index="0"
-              (click)="onSortHandler(field.sortName)" >{{field.columnLabel}}</button>
+      <button class="column-sort" *ngIf="field.columnLabel && field.columnLabel !== 'Assignee'" type="button" [id]="'sort_by_' + field.sortName" data-index="0"
+              (click)="onSortHandler(field.sortName)"> {{field.columnLabel}}</button>
+              <h1 class="assignee" *ngIf="field.columnLabel === 'Assignee'">Assignee:</h1>
```

Comment:
As discussed, this would be made more generic and flexible if ngIf="field.sortable" or similar because currently we can only check for 'Assignee'